contracts_and_coroutines.html

```diff
@@ -217,7 +217,7 @@ <h2> Coroutine properties </h2>
 <h3> Copies of function parameters </h3>
 
 <p> Another property specific to coroutines is that when the coroutine is invoked
-    the function parameters are <em>moved</em> into the coroutine frame
+    the function parameters are <em>copied</em> into the coroutine frame

```

Comment:
They are actually moved. That is, function parameters are left in the moved from state. This follows from http://eel.is/c++draft/dcl.fct.def.coroutine#13.
While this paragraph indeed uses term "copy", in the end it says, "is direct-initialized from an xvalue of type `T` referring to the parameter", which effectively makes the operation a move.

Reply:
perhaps we should just spell this out so that it is clear what the issue is? (we can then use the wording as per the std?)

Reply:
This is a trade-off. "Keep close to the standardese" versus "use simple language". I am generally fine with your suggestion. since you are modifying this part, do you want to suggest the text?

Reply:
yes, it's a trade-off indeed - see how the updated version looks? (i can either squash and force-push this branch or we can just merge two commits .. whatever you prefer)